public_html/wp-content/mu-plugins/send.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Dotorg\Slack;
+
+interface User {
+	public function get_icon();
+	public function get_name();
+}
+
+class Send {
+
+	/**
+	 * The constant overrides the property,
+	 * which is set with the testing() method.
+	 */
+	const testing = false;
+	protected $testing;
+
+	protected $webhook;
+	protected $user;
+	protected $icon;
+	protected $username;
+	protected $text = '';
+	protected $attachments = array();
+
+	function __construct( $webhook ) {
+		$this->webhook = $webhook;
+	}
+
+	function set_user( User $user ) {
+		$this->user = $user;
+	}
+
+	function set_icon( $icon ) {
+		$this->icon = $icon;
+	}
+
+	function get_icon() {
+		if ( $this->icon ) {
+			return $this->icon;
+		} elseif ( $this->user ) {
+			return $this->user->get_icon();
+		}
+		return ':wordpress:';
+	}
+
+	function set_username( $username ) {
+		$this->username = $username;
+	}
+
+	function get_username() {
+		if ( $this->username ) {
+			return $this->username;
+		} elseif ( $this->user ) {
+			return $this->user->get_name();
+		}
+		return 'Bot';
+	}
+
+	function get_text() {
+		return $this->text;
+	}
+
+	function set_text( $text ) {
+		echo 'hello from set text';
```

Comment:
It looks like this was added as a temporary debugging step after copying the file, so it should probably be removed now.

Reply:
```suggestion
```
nice catch. will remove.